Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1098,8 +1098,11 @@ void RenderLayerScrollableArea::computeScrollDimensions()
 {
     m_scrollDimensionsDirty = false;
 
-    m_scrollWidth = roundToInt(overflowRight() - overflowLeft());
-    m_scrollHeight = roundToInt(overflowBottom() - overflowTop());
+    RenderBox* box = m_layer.renderBox();
```

Comment:
Should we check `box` against null ?

Reply:
Added `ASSERT(box)` here to match other member functions.